.github/workflows/gate.yaml

```
@@ -5,6 +5,22 @@ on:
   pull_request:
     branches: [ 'master', 'stabilization*' ]
 jobs:
+  validate-centos7:
+    name: Build, Test on CentOS 7(Container)
+    runs-on: ubuntu-latest
+    container:
+      image: centos:7
+    steps:
+      - name: Install Deps
+        run: yum install -y cmake make openscap-utils PyYAML libxslt expat xml-common python-jinja2
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: ./build_product rhel7
```

```suggestion
        run: ./build_product rhel7 --derivatives
```

since we are building within centos, let's build the centos 7 derivatives as well